src/h5web/guards.ts

```diff
@@ -51,9 +51,15 @@ export function assertOptionalStr(
   }
 }
 
-export function assertNumOrStr(val: unknown): asserts val is number | string {
-  if (typeof val !== 'number' && typeof val !== 'string') {
-    throw new TypeError('Expected number or string');
+export function assertNumOrStrOrBool(
+  val: unknown
+): asserts val is number | string | boolean {
+  if (
+    typeof val !== 'number' &&
+    typeof val !== 'string' &&
+    typeof val !== 'boolean'
+  ) {
+    throw new TypeError('Expected number or string or boolean');
```

Comment:
We could add a new type def for `number | string | boolean`. 

Their common point is the fact that they can easily be converted and displayed as strings so `StringLikeType`, `DisplayableType` ?